src/eduid_common/api/schemas/models.py

```diff
@@ -16,6 +16,8 @@ def ok(self):
     def error(self):
         return 'error'
 
+flux_response_status = FluxResponseStatus()
+
```

Comment:
Instead of instantiating FluxResponseStatus can't we just change it to a dataclass and it will work as intended.

```
@dataclass 
class FluxResponseStatus(): 
    ok = 'ok' 
    error = 'error' 

FluxResponseStatus.error                                                                                                                                                                                                         
'error'

FluxResponseStatus.ok                                                                                                                                                                                                            
'ok'
```

Reply:
yep, nicer :)